bgpd/bgp_nht.c

```diff
@@ -191,7 +192,21 @@ int bgp_find_or_add_nexthop(struct bgp *bgp_route, struct bgp *bgp_nexthop,
 		 * addr */
 		if (make_prefix(afi, pi, &p) < 0)
 			return 1;
-
+		if (orig_prefix) {
+			if ((orig_prefix->family == AF_INET && orig_prefix->prefixlen == 32
```

Comment:
I see we have the same pattern in nexthop_active(), can this be somehow reused?

Reply:
actually, I will use prefix_same() API